ci-operator/config/redhat-chaos/prow-scripts/redhat-chaos-prow-scripts-main__4.18-nightly-krkn-hub-aws.yaml

```diff
@@ -45,4 +45,4 @@ zz_generated_metadata:
   branch: main
   org: redhat-chaos
   repo: prow-scripts
-  variant: 4.18-nightly-component-readiness
+  variant: 4.18-nightly-krkn-hub-aws
```

Comment:
I think the variant should just be `4.18` or `4.18-nightly`

Currently the resulting periodic name is `periodic-ci-redhat-chaos-prow-scripts-main-4.18-nightly-krkn-hub-aws-krkn-hub-component-readiness-418` 

`krkn-hub` is duplicated and aws should be part of the test definition:
 `- as: krkn-hub-component-readiness-418` -> `- as: krkn-hub-aws`

That should result in `periodic-ci-redhat-chaos-prow-scripts-main-4.18-krkn-hub-aws`


I don't think you need `component-readiness` in the periodic name but if you need some differentiator maybe the abbreviated form:
 `- as: krkn-hub-cr-aws`


That would result in `periodic-ci-redhat-chaos-prow-scripts-main-4.18-krkn-hub-cr-aws`

Reply:
@neisw made the changes requested. PTAL when you get time please. Thanks.